docs/sql/float.md

```diff
@@ -99,4 +99,40 @@ REAL '1.23'  -- string style
     <td><code>PI</code></td>
     <td>Returns the approximate value of <code>PI</code> as double. Note that <code>()</code> is not required. Example: <code>SELECT PI;</code></td>
   </tr>
+  <tr>
+    <td><code>TAN(value)</code></td>
+    <td>The tangent of the value as radians. <code>tan</code> only supports arguments of type double, so all other types are cast to double. Returns a double. For undefined values of <code>tan</code> (like <code>tan(pi / 2)</code>) an arbitrary value may be returned.</td>
+  </tr>
+  <tr>
+    <td><code>COT(value)</code></td>
+    <td>The cotangent of the value as radians. <code>cot</code> only supports arguments of type double, so all other types are cast to double. Returns a double. For undefined values of <code>cot</code> (like <code>cot(pi)</code>) an arbitrary value may be returned.</td>
```

Comment:
I don't think this value is in radians, the result is in radians.

Reply:
Yeah, I meant the cotangent is returned as radians, but it could be worded more clearly.

Reply:
yes, confused it with atan